packages/react-native-cli/src/lib/Gradle.ts

```diff
@@ -0,0 +1,100 @@
+import { Logger } from '../Logger'
+import { promises as fs } from 'fs'
+import path from 'path'
+
+const GRADLE_PLUGIN_IMPORT = 'classpath("com.bugsnag:bugsnag-android-gradle-plugin:5.+")'
+const GRADLE_PLUGIN_APPLY = 'apply plugin: "com.bugsnag.android.gradle"'
+const DOCS_LINK = 'https://docs.bugsnag.com/build-integrations/gradle/#installation'
+
+export async function modifyRootBuildGradle (projectRoot: string, logger: Logger): Promise<void> {
+  logger.debug('Looking for android/build.gradle')
+  const topLevelBuildGradlePath = path.join(projectRoot, 'android', 'build.gradle')
+  logger.debug('Importing com.bugsnag:bugsnag-android-gradle-plugin')
+  try {
+    await insertValueAfterPattern(
+      topLevelBuildGradlePath,
+      /[\r\n]\s*classpath\("com.android.tools.build:gradle:\d+\.\d+\.\d+"\)/,
```

Comment:
There are a couple of ways this dependency could be declared, particularly if a user has a brownfield app that they've manually setup themselves. It'd be good to consider the following tweaks as hopefully they'd be relatively little effort and capture some common use cases:

1. Gradle can use either single or double quotes for declaring dependencies, so this could try and check both
2. Gradle dependency versions can be expressed in [various different ways](https://docs.gradle.org/current/userguide/single_versions.html) and can even be a [string interpolated groovy variable](https://github.com/bugsnag/bugsnag-android/blob/master/build.gradle#L15), so it's probably best to assume that anything that comes after `com.android.tools.build:gradle:` is a valid version string